NitroxServer/NitroxServer.csproj

```diff
@@ -14,6 +14,7 @@
     </ItemGroup>
 
     <ItemGroup>
+      <Reference Include="PresentationFramework" />
```

Comment:
This project should not reference UI libraries. Instead, throw exceptions and let the UI display it as a message box.